l10n_cl_etd_account/models/account_invoice.py

```diff
@@ -29,3 +40,26 @@ def invoice_validate(self):
             if sign and invoice.type in ('out_invoice', 'out_refund'):
                 self.with_delay().document_sign()
         return res
+
+    def get_reverse_sii_code(self, code=False):
+        return code and SII_MAPPING.get(code, False) or False
+
+    def get_reverse_sii_document(self):
+        sii_code = self.get_reverse_sii_code(code=self.class_id.code)
+        return self.env['sii.document.class'].search([
+            ('code', '=', sii_code)
+        ], limit=1)
+
+    @api.multi
+    @api.returns('self')
```

Comment:
Why the api.returns decorator? Could it be removed?

Reply:
It is there on the original method...